service-sdk/10.0.0/migrate-to-open-api/messaging-api/typescript-angular-api/message-http-ang.api.md

```diff
@@ -0,0 +1,259 @@
+## API Report File for "@backbase/message-http-ang"
```

Comment:
Should this file be included? It looks like it's the only build artefact - everything else seems to be "source"

Reply:
Isn't that the point? So that from now on you can detect API changes by reviewing PR

Reply:
In an actual project, yes. In this sample, I'm not sure.
Is it an example project showing how a real project looks, or sample code showing the code you need to write in your own project? I'm okay with either, just asking the question :)